synphot/wrapper.py

```diff
@@ -0,0 +1,153 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""This module is a wrapper for synphot's most commonly used functions.
+
+"""
+
+# STDLIB
+import math
+import warnings
+
+# THIRD-PARTY
+import numpy as np
+
+# ASTROPY
+from astropy import log
+from astropy import units as u
+from astropy.utils.exceptions import (AstropyUserWarning,
+                                      AstropyDeprecationWarning)
+from astropy.modeling.models import Const1D
+
+# LOCAL
+from . import binning, exceptions, units, utils
+from .models import Empirical1D
+from .spectrum import BaseSourceSpectrum, SourceSpectrum, SpectralElement
+from .observation import Observation, ccd_snr, exptime_from_ccd_snr
+
+
+class Observe:
```

Comment:
Same comment as earlier, prefer `class Observe(object):` over `class Observer`. 

***

Sidebar: perhaps this should be `Expose` rather than `Observe` since synphot already has an `Observation` class.